### PR TITLE
refactored

### PR DESCRIPTION
**Pull request by MrBushidoBrownn**
Let me know what you think JJ

### CODE REVIEW

File: base/game_events/GameEventManager.ts [hunk 1]
```diff
@@ -182,7 +182,7 @@ export class GameEventManager {
             voice_key: npc.voice_key,
             hero_direction: this.data.hero.current_direction,
         });
-        this.fire_next_step = dialog_manager.next.bind(dialog_manager, async finished => {
+        this.fire_next_step = dialog_manager.next.bind(dialog_manager, async (finished: any) => {
```
Comment (jjppof) [inline base/game_events/GameEventManager.ts:185]: change this type to `boolean`.

**Review by jjppof — CHANGES_REQUESTED** [review]
Beyond that comment, always before your last commit, run `npm run format` in order to format the things you changed. This is necessary to merge your code with `master`.

In order to submit the new changes, just commit and push in the same branch. Your PR will update automatically.

File: base/game_events/GameEventManager.ts [hunk 2]
```diff
@@ -182,7 +182,7 @@ export class GameEventManager {
             voice_key: npc.voice_key,
             hero_direction: this.data.hero.current_direction,
         });
-        this.fire_next_step = dialog_manager.next.bind(dialog_manager, async (finished: any) => {
+        this.fire_next_step = dialog_manager.next.bind(dialog_manager, async (finished: !!any) => {
```
Comment (MrBushidoBrownn) [inline base/game_events/GameEventManager.ts:185]: @jjppof let me know if this works

Comment (jjppof) [reply base/game_events/GameEventManager.ts:185]: No, do this: `finished: boolean`. Remove the `any`.

**Pull request merged**